presto-main/src/main/java/io/prestosql/sql/planner/LocalDynamicFiltersCollector.java

```diff
@@ -20,8 +20,11 @@
 import io.prestosql.spi.connector.ColumnHandle;
```

Comment:
please add support to `DynamicFilterService` too

Reply:
Added `DynamicFilterService` support.

Reply:
Just to be sure - I had to add the following line to support cross-joins:
https://github.com/prestosql/presto/blob/6323d03a2ed123a77f5e62415f5d3b8a7c7c7552/presto-main/src/main/java/io/prestosql/operator/JoinUtils.java#L71
Is it OK?